src/chains/views.py

```diff
@@ -0,0 +1,20 @@
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework.generics import ListAPIView
+from rest_framework.pagination import LimitOffsetPagination
+
+from .models import Chain
+from .serializers import ChainSerializer
+
+
+class ChainsListView(ListAPIView):
+    serializer_class = ChainSerializer
+    pagination_class = LimitOffsetPagination
+    pagination_class.max_limit = 10
+    pagination_class.default_limit = 10
+
+    @swagger_auto_schema()
```

Comment:
Do you need to do this to get the schema? It should be automatic without overriding `get`